### DISCUSSION TITLE
Miryoku ZMK: Add combos involving thumb keys

### DISCUSSION THREAD

**Discussion by davidsteinberger** [manna-harbour/miryoku#305] [Q&A] [answered]
I'm quite familiar with Miryoku on QMK but just started building my first keyboard with ZMK. 
I use more combos in my layout. I started to convert the combos from my QMK setup, however all combos that involve the thumb keys do not work for me. Is there a possibility to add those?

**Comment by manna-harbour** [comment, marked as answer]
Combos on the thumbs work the same way as any other key positions. However, if you already have https://github.com/manna-harbour/miryoku_zmk#key-emulation-combos enabled (thumb combos are enabled automatically on 34 key boards etc.) you'll have to increase the combos per key kconfig, and also note that key positions are based on keyboard keymap key position index, e.g. in https://github.com/manna-harbour/miryoku_zmk/blob/master/miryoku/mapping/50/kyria.h you need to count in this section https://github.com/manna-harbour/miryoku_zmk/blob/aa47838943624650f51830df99faf217e2eb48f3/miryoku/mapping/50/kyria.h#L28-L31 as shown here https://github.com/manna-harbour/miryoku_zmk/blob/aa47838943624650f51830df99faf217e2eb48f3/miryoku/mapping/50/kyria.h#L32-L33.
